themes/classic/templates/checkout/_partials/order-final-summary-table.tpl

```diff
@@ -0,0 +1,9 @@
+{extends file='checkout/_partials/order-confirmation-table.tpl'}
+
+{block name='order-items-table-head'}
+<div id="order-items" class="col-md-12">
+  <h3 class="card-title h3">
+	{$products_count} {if $products_count == 1}{l s=' Item' d='Shop.Theme.Checkout'}{else}{l s=' Items' d='Shop.Theme.Checkout'}{/if} {l s=' in your cart' d='Shop.Theme.Checkout'}
```

Comment:
We'd like to avoid strings like this, it makes it hard to understand for translators.
Please use something like

```smarty
{if $products_count == 1}
  {l s='%s item in your cart' sprintf=$products_count d='Shop.Theme.Checkout'}
{else}
  {l s='%s items in your cart' sprintf=$products_count d='Shop.Theme.Checkout'}
{/if}
```